docs/authoring/servertaskauthoring.md

```diff
@@ -95,11 +95,11 @@ Let us understand few concepts/tips & tricks.
   
 
 #### Built-in server tasks making http requests : -
-  [InvokeRestApi](https://github.com/Microsoft/vsts-tasks/blob/master/Tasks/InvokeRestApi/task.json)  
-  [AzureMonitor](https://github.com/Microsoft/vsts-tasks/blob/master/Tasks/AzureMonitor/task.json)   
-  [AzureFunction](https://github.com/Microsoft/vsts-tasks/blob/master/Tasks/AzureFunction/task.json)   
-  [QueryWorkItems](https://github.com/Microsoft/vsts-tasks/blob/master/Tasks/QueryWorkItems/task.json)   
+  [InvokeRestApiV1](https://github.com/Microsoft/vsts-tasks/blob/master/Tasks/InvokeRestApiV1/task.json)  
```

Comment:
You can keep names same. Use InvokeRestApi instead of  InvokeRestApiV1. Same for other tasks

Reply:
Done